templates/ts-apollo-mongodb-backend/src/index.ts

```diff
@@ -1,55 +1,54 @@
 // eslint-disable-next-line @typescript-eslint/no-require-imports
-require('dotenv').config()
-import path from 'path'
-import { ApolloServer } from "apollo-server-express"
-import { buildGraphbackAPI } from 'graphback'
-import { loadSchemaSync } from '@graphql-tools/load'
-import { GraphQLFileLoader } from '@graphql-tools/graphql-file-loader'
-import { createMongoDbProvider } from '@graphback/runtime-mongo'
-import cors from "cors"
-import express from "express"
-import http from "http"
+require('dotenv').config();
+import { ApolloServer } from 'apollo-server-express';
+import { buildGraphbackAPI } from 'graphback';
+import { createMongoDbProvider } from '@graphback/runtime-mongo';
+import cors from 'cors';
+import express from 'express';
+import http from 'http';
 import { loadConfigSync } from 'graphql-config';
-import { connectDB } from './db'
-import { noteResolvers } from './resolvers/noteResolvers'
+import { connectDB } from './db';
+import { noteResolvers } from './resolvers/noteResolvers';
 
 async function start() {
+  const app = express();
 
-  const app = express()
-
-  app.use(cors())
+  app.use(cors());
 
   const graphbackExtension = 'graphback';
-  const config = loadConfigSync({ extensions: [
-    () => ({
-      name: graphbackExtension
-    })
-  ]});
-  const projectConfig = config.getDefault()
+  const config = loadConfigSync({
+    extensions: [
+      () => ({
+        name: graphbackExtension,
+      }),
+    ],
+  });
+
+  const projectConfig = config.getDefault();
   const graphbackConfig = projectConfig.extension(graphbackExtension);
 
-  const modelDefs = config.getDefault().loadSchemaSync(graphbackConfig.model);
+  const modelDefs = projectConfig.loadSchemaSync(graphbackConfig.model);
 
-  const db = await connectDB()
+  const db = await connectDB();
 
   const { typeDefs, resolvers, contextCreator } = buildGraphbackAPI(modelDefs, {
-    dataProviderCreator: createMongoDbProvider(db)
+    dataProviderCreator: createMongoDbProvider(db),
   });
 
   const apolloServer = new ApolloServer({
     typeDefs,
     resolvers: [resolvers, noteResolvers],
-    context: contextCreator
-  })
+    context: contextCreator,
```

Comment:
Does your editor prefer trailing commas? Not that it matters, they are useful!

Reply:
Ah this should not have been committed. I ran `prettier` on the wrong project.